app/controllers/activities_controller.rb

```diff
@@ -0,0 +1,76 @@
+class ActivitiesController < ApplicationController
+  before_action :logged_in_user, only: %i(create destroy)
+  before_action :correct_user, only: %i(edit update)
+  before_action :find_post, only: %i(create destroy update)
+
+  def index
+    @activities = Activity.all
+  end
+
+  def new
+    @activities = Activity.new
```

Comment:
with this case, Activity.new will create AN objeck. therefor you should use @activity
Using plurality when return > 1 record